openmetadata-service/src/main/java/org/openmetadata/service/security/mask/PIIMasker.java

```diff
@@ -137,6 +140,22 @@ public static Table getTableProfile(Table table) {
     return table;
   }
 
+  public static List<ColumnProfile> getColumnProfile(
+      String fqn, List<ColumnProfile> columnProfiles) {
+    Table table =
+        Entity.getEntityByName(
+            Entity.TABLE, FullyQualifiedName.getTableFQN(fqn), "columns,tags", Include.ALL);
+    Column column =
+        table.getColumns().stream()
+            .filter(c -> c.getFullyQualifiedName().equals(fqn))
+            .findFirst()
+            .orElse(null);
+    if (column != null && hasPiiSensitiveTag(column)) {
+      return Collections.nCopies(columnProfiles.size(), new ColumnProfile());
```

Comment:
Thinking here is to keep a consistent API, return null objects (vs null data). Let me know what you think